lib/org/openpsa/sales/handler/render.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package org.openpsa.sales
+ * @author The Midgard Project, http://www.midgard-project.org
+ * @copyright The Midgard Project, http://www.midgard-project.org
+ * @license http://www.gnu.org/licenses/lgpl.html GNU Lesser General Public License
+ */
+
+/**
+ * Sales action handler ***creates pdf-file***
+ *
+ * @package org.openpsa.sales
+ */
+class org_openpsa_sales_handler_render extends midcom_baseclasses_components_handler
+{
+    private $salesproject =null;
+
+    private $client_class =null;
```

Comment:
there's a whitesapce missing after `=`. But you can also just remove the `null` assignment, this is done automatically by PHP, so 

```
private $client_class;
``` 

is sufficient